app/controllers/admin/events_controller.rb

```diff
@@ -50,8 +50,8 @@ def show
       @ratings = @event.votes.includes(:user)
       @difficulty_levels = @program.difficulty_levels
       @versions = @event.versions |
-       PaperTrail::Version.where(item_type: 'Commercial').where_object(commercialable_id: @event.id, commercialable_type: 'Event') |
-       PaperTrail::Version.where(item_type: 'Commercial').where_object_changes(commercialable_id: @event.id, commercialable_type: 'Event') |
+       PaperTrail::Version.where(item_type: 'Commercial').where('object LIKE ?', "%commercialable_id: #{@event.id}\ncommercialable_type: Event%") |
```

Comment:
Because that is how paper trail stores object. Something like this :

>"---\nid: 5\ncommercial_id: \ncommercial_type: \ncommercialable_id : 8\ncommercialable_type: Event\ncreated_at: 2017-03-06 19:40:15.067701000 Z\nupdated_at: 2017-03-06 19:40:15.067701000 Z\nurl: https://www.youtube.com/watch?v="